### ISSUE TITLE
About use epoch based learning rate scheduler

### ISSUE THREAD

**Issue by LinpengPan** [chaofengc/IQA-PyTorch#39]
Does this method support CosineAnnealingRestartLR？
